manage.py

```diff
@@ -198,6 +247,66 @@ def set_database_configuration(
         )
 
 
+@manager.command
+@manager.option("-d", "--default_database", help="Default data source")
+def set_default_datasource(default_database=None):
+    if default_database:
+        update_config_file({"DEFAULT_DATASOURCE": default_database})
+    else:
+        search_omero_app.logger.info("No attribute is provided")
+
+
+@manager.command
+@manager.option("-n", "--name", help="data source name")
+@manager.option(
+    "-i",
+    "--images_folder",
+    help="path to a folder containing CSV files which contains the image data ",
+)
+@manager.option(
+    "-p", "--projects_file", help="path to a file containing the projects data"
+)
+@manager.option(
+    "-s", "--screens_file", help="path to a file containing the screens data"
+)
+@manager.option("-d", "--datasource_type", help=" data source type; supports CSV")

```

Comment:
It seems that ``-d`` is used and abused in all the commands
Should we use for that one ``-t`` for type

Reply:
Ok, I will change it to `-s`